packages/cli/test/helpers/prepare.js

```diff
@@ -415,6 +415,23 @@ module.exports = async function prepare(session, binaryPath, tmpFixturesDir) {
         projectId: 'QmRoBYhejkkmssotLZr8tWgewPdPcjYucYUNERFbhJrRNi',
       }),
     },
+    'vc-build-speed-insights': {
+      '.vercel/project.json': JSON.stringify({
+        orgId: '.',
+        projectId: '.',
+        settings: {
+          framework: null,
+        },
+      }),
+      'package.json': JSON.stringify({
+        scripts: {
+          build: 'mkdir -p public && echo hi > public/index.txt',
+        },
+        dependencies: {
+          '@vercel/speed-insights': 'latest',
```

Comment:
Does this get installed? If so, lets use a specific version here

Reply:
yeah, good call; updated